deploy

```diff
@@ -0,0 +1 @@
+config
```

Comment:
<!--whitespace-bot-->
```suggestion
config

```

Reply:
We can ignore this, since this is output of ln -s. We cannot add empty, because it breaks the symbolic link.